packages/server-nodejs/src/JitarServer.ts

```diff
@@ -51,37 +61,40 @@ export default class JitarServer
         this.#app.use(express.urlencoded({ extended: true }));
 
         this.#app.disable('x-powered-by');
+
+        this.#options = ServerOptionsReader.read();
+        this.#configuration = RuntimeConfigurationLoader.load(this.#options.config);
+        this.#logger = LogBuilder.build(this.#options.loglevel);
+    }
+
+    async build(): Promise<void>
+    {
+        this.#runtime = await RuntimeConfigurator.configure(this.#configuration);
+        this.#addControllers(this.#configuration, this.#runtime, this.#logger);
```

Comment:
We only pass instance fields, so we can remove them here and pick them up where they are used.